common/rfb/pam.c

```diff
@@ -24,10 +24,13 @@
 
 #include <stdlib.h>
 #include <string.h>
+#include <syslog.h>
 #include <security/pam_appl.h>
 
 #include <rfb/pam.h>
 
+char displayStr[32]; /* set by Xvnc and x0vncserver */
```

Comment:
Given how we handle similar things, I'd say the most appropriate place is for this to be a static member of a class. `UnixPasswordValidator` is the wrapper for this functionality, so let's put it there.

Reply:
Not sure if we can pass a static member of a C++ class to the C code in libpam without resorting to come trick, because we need to pass `pam_callback()` to `pam_start()` in a `pam_conv`struct, but I will investigate this approach.